### PR TITLE
[CSL-2389] Adds option to consume library as a bundle

### PR DESCRIPTION
**Pull request by Mudaafi**
Users may now consume this library as a bundle via two ways:

#### Via NPM
`npm i '@constructor-io/constructorio-ui-autocomplete'`
```
import CioAutocomplete from '@constructor-io/constructorio-ui-autocomplete/constructorio-ui-autocomplete-bundled';

CioAutocomplete({
  selector: '#autocomplete-container',
  includeCSS: true, // Include the default CSS styles. Defaults to true.
  apiKey: '',
  onSubmit: (submitEvent) => console.dir(submitEvent),
  // ... additional arguments
});
```

#### Via Script Tags (only locally for now)
`<script type="module" src="./constructorio-ui-autocomplete-bundled.js"></script>`
```
window.CioAutocomplete({
  selector: '#autocomplete-container',
  includeCSS: true, // Include the default CSS styles. Defaults to true.
  apiKey: '',
  onSubmit: (submitEvent) => console.dir(submitEvent),
  // ... additional arguments
});
```

### CODE REVIEW

**Review by esezen — APPROVED** [review]
This is looking good. Let's get one more review before we merge

File: README.md
Comment (Mudaafi) [reply README.md:126]: Sounds good

File: README.md
Comment (Mudaafi) [reply README.md:123]: ooh nice eye, will fix

**Review by stanlp1 — APPROVED** [review]
LGTM!

**Review by jjl014 — APPROVED** [review]
LGTM! 🚀

**Pull request merged**
